flutter/pubspec.yaml

```diff
@@ -19,6 +19,7 @@ dependencies:
   package_info_plus: '>=1.0.0 <=5.0.1'
   meta: ^1.3.0
   ffi: ^2.0.0
+  connectivity_plus: ^5.0.2
```

Comment:
This is kinda against https://develop.sentry.dev/philosophy/#dependencies-cost and as known from package_info_plus it's a quite frequent source of major updates.